spec/requests/authorized_family_members_requests_spec.rb

```diff
@@ -0,0 +1,57 @@
+require "rails_helper"
+
+RSpec.describe "Authorized Family Members Controller", type: :request do
+  let!(:partner) { create(:partner) }
+  let!(:user) { create(:user, partner: partner) }
```

Comment:
I believe you could get rid of the bang on both `partner` and `user` creation once `before` block is invoking `user ` creation and by consequence creating the partner, also the `family` bang is creating the partner.

It's not a big deal actually :nail_care: